### ISSUE TITLE
Panic with blank identifier

### ISSUE THREAD

**Issue by mirshahriar** [gophersbd/gouberfmt#8]
```
data := []int{1, 2, 3}
_ = data
``` 
```
var anotherIfacePointer *interface{}
_ = anotherIfacePointer
```
This [part](https://github.com/gophersbd/gouberfmt/blob/994c457e690e3e939c96e6c73763452ada5a702f/analyzers/copyboundary/copy_boundary.go#L83) causes panic. @faem

**Comment by faem** [comment]
Please, Close this? @sadlil bhai, @aerokite bhai

**Comment by mirshahriar** [comment]
resolved in #9
